crates/gpui/src/elements/text.rs

```diff
@@ -244,6 +244,8 @@ struct TextLayoutInner {
     bounds: Option<Bounds<Pixels>>,
 }
 
+static ELLIPSIS: &str = "…";
```

Comment:
Does this need to be a `static`, or can we make it a `const`?

```suggestion
const ELLIPSIS: &str = "…";
```